Base/QTGUI/qSlicerApplication.cxx

```diff
@@ -828,11 +828,13 @@ void qSlicerApplication::displayApplicationInformations() const
   numberOfPhysicalCPU = (unsigned int) info.dwNumberOfProcessors;
 #endif
 
-  qDebug("%s: %s %.3f MHz, %d cores",
+  unsigned int numberOfLogicalCPU = systemInfo->GetNumberOfLogicalCPU();
+
+  qDebug("%s: %s %s, %d cores, %d logical processors",
          qPrintable(titles.at(4).leftJustified(titleWidth, '.')),
          systemInfo->GetVendorString() ? systemInfo->GetVendorString() : "unknown",
-         systemInfo->GetProcessorClockFrequency()/1000,
-         numberOfPhysicalCPU);
+         systemInfo->GetModelName() ? systemInfo->GetModelName() : "unknown",
+         numberOfPhysicalCPU, numberOfLogicalCPU);
```

Comment:
Got it ... it was mention in the commit mesg. 

On your machine, does this correctly describe the before and after ?  Could you then amend the last commit including it ?

```
Before:
CPU ......................: GenuineIntel 2.300 MHz, 1 cores

After:
CPU ......................: GenuineIntel Intel(R) Xeon(R) CPU E5-2630 0 @ 2.30GHz, 12 cores, 24 logical processors
```